platform-dependent-components/problem-solver/cxx/generate-response-module/agent/GenerateResponseAgent.cpp

```diff
@@ -69,8 +75,9 @@ void GenerateResponseAgent::attachAnswer(
     ScAddr const & messageAddr,
     ScAddr const & answerAddr)
 {
-  utils::GenerationUtils::generateRelationBetween(&m_memoryCtx, messageAddr, messageAnswer, Keynodes::nrel_reply_structure);
-  m_memoryCtx.CreateEdge(ScType::EdgeAccessConstPosTemp, answerAddr, messageAnswer);
+  utils::GenerationUtils::generateRelationBetween(
+      &m_memoryCtx, messageAddr, messageAnswer, Keynodes::nrel_reply_structure);
+  m_memoryCtx.CreateEdge(ScType::EdgeAccessConstPosPerm, answerAddr, messageAnswer);
```

Comment:
Why do you need it?

Reply:
answerAddr is structuresSet in structure translation agent, which must contain a structure (messageAnswer) for translation agent to find and translate